.rubocop.yml

```diff
@@ -30,3 +30,7 @@ Style/IndentationConsistency:
 
 Style/RedundantReturn:
   Enabled: true
+
+Style/StringLiterals:
+  Enabled: true
+  EnforcedStyle: single_quotes
```

Comment:
Every Ruby code base I've worked on largely follows this rule: single quotes for strings without interpolation, double quotes for strings with. I believe that's the prevailing style. But I can adapt to whatever, I just want it to be consistent. The hardest thing about working with the specs is sometimes they're single-quoted, sometimes double, and you have manually fix that if you ever want to run the snippet in a shell.